compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/NodeCloner.java

```diff
@@ -2286,4 +2312,12 @@ public void visit(BLangObjectConstructorExpression source) {
 
         source.cloneRef = clone;
     }
+
+    public int getCloneAttempt() {
```

Comment:
I think we can remove these methods now (both get and set)

Reply:
Will remove those